packages/sdk/src/subgraph.ts

```diff
@@ -295,6 +295,94 @@ export class Subgraph {
     return all;
   }
 
+  async getHistoricalTransactions(): Promise<CrosschainTransaction[]> {
```

Comment:
This should likely include any that were cancelled as well, not simply ones that were fulfilled

Reply:
The idea was to do that in steps, starting with resolved only. Let me see if it's easy enough to get in here.

Reply:
Done.